### PR TITLE
docs: add table to prioritize scorecard implementation

### PR DESCRIPTION
**Pull request by inigomarquinez**
**Description**

Based on an Excel sheet of @UlisesGascon with all the packages of the express ecosystem (updated 1st February 2024), I have divided them into 5 blocks of 10 repositories each (plus a block of repositories with low priority as they may be deprecated and another block of repositories that may not be the focus of the scorecard, as they are used for documentation) to prioritize the OpenSSF score card implementation.

For that, I have sorted the repositories by the number of downloads per week and assigned them a priority between 1 (the most important) and 5 (the least important) in the `Scorecard priority block` column. All those repositories with low priority have a priority of 6.

The table is then sorted alphabetically.

This is an initial version, so feel free to add your feedback.

**Context** 

As part of the [OpenSSF Scorecard implementation](https://github.com/expressjs/security-wg/issues/2), the first step will be defining the scope and priority of each repo so we can divide efforts when adding the scorecard to each repo.

### CODE REVIEW

File: tools/scope/packages-in-scope.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+| https://github.com/pillarjs/send                | 30.4                          | 2y               | 781       | 2                            |
+| https://github.com/jshttp/accepts               | 30.2                          | 2y               | 251       | 2                            |
+| https://github.com/jshttp/range-parser          | 29.7                          | 5y               | 91        | 2                            |
+| https://github.com/expressjs/serve-static       | 29.6                          | 2y               | 1400      | 2                            |
+| https://github.com/jshttp/content-type          | 29.1                          | 1y               | 126       | 2                            |
+| https://github.com/jshttp/vary                  | 28.8                          | 6y               | 59        | 2                            |
+| https://github.com/jshttp/methods               | 28.6                          | 8y               | 177       | 2                            |
+| https://github.com/expressjs/express            | 28.23                         | 1y               | 62900     | 2                            |
```
Comment (UlisesGascon) [inline tools/scope/packages-in-scope.md:21]:
```suggestion
| https://github.com/expressjs/express            | 28.23                         | 1y               | 62900     | 1                            |
```

I will suggest an exception for express as we started already to work on it (priority 1)

**Review by UlisesGascon — APPROVED** [review]
LGTM! Thanks for consolidate it @inigomarquinez 🙌

**Comment by wesleytodd** [comment]
Did we cross reference this with the this? https://github.com/expressjs/Admin/pull/1

File: tools/scope/packages-in-scope.md (hunk 1 above)
Comment (inigomarquinez) [reply tools/scope/packages-in-scope.md]: Cool. I've switched priority with https://github.com/jshttp/content-disposition, to have equal blocks of 11 packages each.

Comment (UlisesGascon) [reply tools/scope/packages-in-scope.md]: Thanks!

**Comment by carpasse** [comment]
@inigomarquinez I've done the cross reference agains the repos listed in https://github.com/expressjs/Admin/pull/1
Here are my findings:
[packages-in-scope.md](https://github.com/expressjs/security-wg/files/14589238/packages-in-scope.md)

Possible improvements to the scope table

1. Create a table per org.
2. Add a project name column
3. Sort table rows by project name. It makes everything easier to find

P.S. @wesleytodd I think some of the repos listed this PR are not listed in https://github.com/expressjs/Admin/pull/1 I've listed them in the [file](https://github.com/expressjs/security-wg/files/14589238/packages-in-scope.md) with my findings in case you want to take a look

**Comment by inigomarquinez** [comment]
> 77ee84c

@carpasse , thanks for the idea. I've refactored the file to include the information you provide in the file in [this commit](https://github.com/expressjs/security-wg/pull/10/commits/77ee84c42925e994405f57f92cb1aad0d5461506)

@wesleytodd , this should solve your comment.

**Review by marco-ippolito — APPROVED** [review]

**Review by wesleytodd — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]
LGTM! Thanks @inigomarquinez

**Pull request merged**